src/common/Buttons/PrimaryButton.tsx

```diff
@@ -16,7 +16,7 @@ interface PrimaryButtonProps extends ButtonHTMLAttributes<HTMLButtonElement> {
 	onEmitAnalytics?: ActionButtonsProps["onEmitAnalytics"];
 }
 
-const PrimaryButton: FC<PrimaryButtonProps> = props => {
+const PrimaryButton = forwardRef<HTMLButtonElement, PrimaryButtonProps>((props, ref) => {
```

Comment:
do we maybe need the same logic on SecondaryButton component?

Reply:
Right now it is not necessary, but I can add that.

Reply:
I would suggest to add that in this PR, otherwise later we can easily miss that detail.